cytopipe/cli/setup_init.py

```diff
@@ -0,0 +1,90 @@
+"""setup_init.py
+
+Contains functions for initializing different data structures 
+for processing
+"""
+
+from typing import Union
+from pathlib import Path
+
+
+def init_cp_data(data_fp: Union[list[str], str], metadata_fp: str, barcode_fp: str):
+    """Sets up directory for Cell Profiler datasets
```

Comment:
CellProfiler as one word please.

Also, can you elaborate a bit more (like 1 or two more sentences) on the nature of this function? I think it's worth mentioning the creation of `symlinks` and briefly touching upon what the storage implications are.